### PR TITLE
Создание приложения.

### PR DESCRIPTION
**Pull request by Gennadievi4**

### CODE REVIEW

File: Guess.Yourself/CommandBase.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+{
+    public abstract class CommandBase : ICommand
+    {
+        public event EventHandler CanExecuteChanged
+        {
+            add => CommandManager.RequerySuggested += value;
+            remove => CommandManager.RequerySuggested -= value;
+        }
+
+        public abstract bool CanExecute(object parametr);
```
Comment (artemfomin) [inline Guess.Yourself/CommandBase.cs]: Typo in parameter

File: Guess.Yourself/CommandBase.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+        public abstract void Execute(object parametr);
```
Comment (artemfomin) [inline Guess.Yourself/CommandBase.cs]: Same typo

File: Guess.Yourself/CommandBase.cs [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+using System.Windows.Input;
+
+namespace Guess.Yourself
+{
+    public abstract class CommandBase : ICommand
+    {
+        public event EventHandler CanExecuteChanged
+        {
+            add => CommandManager.RequerySuggested += value;
+            remove => CommandManager.RequerySuggested -= value;
```
Comment (artemfomin) [inline Guess.Yourself/CommandBase.cs]: add =>
remove => 
Is not a standard coding style. Look for alternatives

File: Guess.Yourself/CommandBase.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Windows.Input;
+
+namespace Guess.Yourself
+{
+    public abstract class CommandBase : ICommand
```
Comment (artemfomin) [inline Guess.Yourself/CommandBase.cs]: public members should have summary

File: Guess.Yourself/DeviceManager.cs [hunk 5]
```diff
@@ -0,0 +1,39 @@
+﻿using RLib;
+using System;
+
+namespace Guess.Yourself
+{
+    class DeviceManager
```
Comment (artemfomin) [inline Guess.Yourself/DeviceManager.cs]: Access modifier should be set explicitly

File: Guess.Yourself/DeviceManager.cs [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+    {
+        readonly UsbWatcher usbWatcher = new UsbWatcher();
```
Comment (artemfomin) [inline Guess.Yourself/DeviceManager.cs]: Access modifier should be set explicitly

File: Guess.Yourself/DeviceManager.cs [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+        public VotumDevicesManager votumManager { get; private set; }
```
Comment (artemfomin) [inline Guess.Yourself/DeviceManager.cs:9]: Public member of private class. Very funny

File: Guess.Yourself/DeviceManager.cs [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            votumManager = manager;
+            InitVotumDevice();
+        }
+
+        public void InitVotumDevice()
+        {
+            votumManager.Settings.InteractiveRemotesSettings.InteractiveMode = true;
+            votumManager.Settings.InteractiveRemotesSettings.AutoT2 = true;
+            usbWatcher.DeviceInserted += Watcher_DeviceInserted;
```
Comment (artemfomin) [inline Guess.Yourself/DeviceManager.cs:20]: Here you add handlers for some events. Where do you remove them?

File: Guess.Yourself/DeviceManager.cs [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+            usbWatcher.DeviceRemoved += Watcher_DeviceInserted;
+            votumManager.Start();
+        }
+
+        private void StartStopDeviceManager()
```
Comment (artemfomin) [inline Guess.Yourself/DeviceManager.cs]: This is called restart by logic

File: Guess.Yourself/MainWindowViewModel.cs [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+using System.Linq;
+using System.Windows.Threading;
+
+namespace Guess.Yourself
+{
+    public class MainWindowViewModel
+    {
+        public ObservableCollection<StudentModel> Students { get; set; } = new ObservableCollection<StudentModel>();
+
+        readonly DeviceManager deviceManager = new DeviceManager(new VotumDevicesManager());
```
Comment (artemfomin) [inline Guess.Yourself/MainWindowViewModel.cs]: Access modifier

File: Guess.Yourself/QuestionView.xaml.cs [hunk 11]
```diff
@@ -0,0 +1,14 @@
...
+using System.Windows;
+
+namespace Guess.Yourself
+{
+    public partial class QuestionView : Window
+    {
+        public QuestionView(List<string> questions)
+        {
+            InitializeComponent();
+            lv.ItemsSource = questions;
```
Comment (artemfomin) [inline Guess.Yourself/QuestionView.xaml.cs]: What the hell is lv?

File: Guess.Yourself/StudentModel.cs [hunk 12]
```diff
@@ -0,0 +1,65 @@
...
+    public class StudentModel : INotifyPropertyChanged
+    {
+        public string Character { get; set; }
+        public string Question { get; set; }
+        public int ReceiverId { get; set; }
+        public int RemoteId { get; set; }
+        public List<string> Questions { get; set; } = new List<string> { "Первый вопрос", "Второй вопрос", "Третий вопрос" };
+        public SolidColorBrush AnswerColor { get; set; }
+
+        AnswerType userAnswer;
```
Comment (artemfomin) [inline Guess.Yourself/StudentModel.cs]: Access modifier and what thell is it?

File: Guess.Yourself/QuestionView.xaml.cs (hunk 11 above)
Comment (Gennadievi4) [reply Guess.Yourself/QuestionView.xaml.cs]: lv -  is the name of ListBox

**Pull request merged**